keystone.js

```diff
@@ -0,0 +1,65 @@
+// Simulate config options from your production environment by
+// customising the .env file in your project's root folder.
+require('dotenv').config();
+
+// Require keystone
+var keystone = require('keystone');
+var handlebars = require('express-handlebars');
+
+// Initialise Keystone with your project's configuration.
+// See http://keystonejs.com/guide/config for available options
+// and documentation.
+
+keystone.init({
+	'name': 'Midwest PHP',
+	'brand': 'Midwest PHP',
+
+	// 'sass': 'public',
```

Comment:
Do you plan to keep the sass or not?  If so then the commented out stuff is fine but otherwise remove.

Reply:
Nope, will yank that.

Reply:
Removed the commented out code